src/pages/day-view.tsx

```diff
@@ -1,11 +1,27 @@
+import clsx from "clsx";
+import { format, isAfter, isBefore } from "date-fns";
+import _ from "lodash";
 import type { NextPage } from "next";
 import { TodoItem } from "../features/todo/list";
 import { Loader } from "../features/ui/loader";
 import { Title } from "../features/ui/title";
 import { trpc } from "../utils/trpc";
-import _ from "lodash";
 
 const DayViewPage: NextPage = () => {
+    return (
+        <main className="grid grow grid-cols-4 divide-x-2 divide-neutral-700">
+            <div className="col-span-3 pr-6 pt-4">
+                <TodosSection />
+            </div>
+
+            <div className="pt-4">
+                <FocusSection />
+            </div>
+        </main>
+    );
+};
```

Comment:
This part is not reused, even though multiple pages use the same structure, it's a good idea to extract it to a component in future